src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -284,10 +286,11 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
 
 for scope in ['disk', 'snapshot']:
     register_extra_cli_argument(scope + ' create', 'source', validator=process_disk_or_snapshot_create_namespace,
-                                help='source to create the disk from, including a sas uri to a blob, managed disk id or name, or snapshot id or name')
+                                help='source to create the disk from, including a sas blob uri to a blob, managed disk id or name, or snapshot id or name')
     register_cli_argument(scope, 'source_blob_uri', ignore_type)
     register_cli_argument(scope, 'source_disk', ignore_type)
     register_cli_argument(scope, 'source_snapshot', ignore_type)
+    register_cli_argument(scope, 'source_storage_account_id', help='used when source blob is in a different subscription')
```

Comment:
Right.  So, do you have to have access rights to that subscription to create a disk from it?  Does it matter if the blob is public or private?